### DISCUSSION TITLE
Red and Green PixelNoise texture?

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#360] [Answers & Questions] [answered]
Guys I am really struggling with understanding one of the concepts in the 'Vertex Textures - Point Clouds and Mesh Instancing from Textures' [video](https://www.youtube.com/watch?v=PrkdnENo8wQ). 

Especially how at around 21:50, he talks about positioning the cubes according to that topmost PixelNoise texture that has a wave gradient (red to green). I am unable to reproduce that smooth looking VizTexture. For me the PixelNoise  Op still produces the random RGB spots.

The settings of the second WaveformGradient, with an 'add' blend mode, are never shown, so maybe that has the explanation? Actually, all 3 confuse me: how he restricted the second VizTexture to only have a green color and the third/bottom one with a smooth white to black gradient, rotating one.

Any help?
![VizTexture](https://user-images.githubusercontent.com/94358787/192631732-818a97ad-8e43-47f6-9042-8f243ac7e090.jpg)

Eagerly awaiting to watch the Cables.gl twitch stream tomorrow for the first time. In fact I created a Twitch and Discord account only to watch/discuss Cables.Gl. :D

**Comment by monolithMktg** [comment]
Ok this is the best I could manage till now but nothing like what is shown in the video:

![VizTexture](https://user-images.githubusercontent.com/94358787/192752002-baac59fe-74b3-455c-bc1f-4c04eafef952.jpg)

**Comment by pandrr** [comment]
turn transparency of in the new imagecompose op, set it to 1

**Reply by monolithMktg** [reply]
Thank you. Raising 'A' value to 1 fixes the transparent background checkerboard on VixTexture, but still the pixel noise is not sharing the blocks to be used in these wave gradients.

**Comment by monolithMktg** [comment, marked as answer]
I think Pandrr's solution in yesterday's live session fixed the issue.
